docs/data/features.yaml

```diff
@@ -0,0 +1,7 @@
+- OCI Image Spec support
```

Comment:
Looks like a map expected here `cannot unmarshal !!seq into map[string]interface {}`

Reply:
Weird. That works just fine locally. I'll look into it.

Reply:
Works locally for me as well, it appears to be running the command I set on the project rather than from the toml. I don't know what version of hugo it is running.